db/migrations/00003_add-account-locale.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE auth.accounts
+  ADD COLUMN locale VARCHAR(2),
+  ALTER COLUMN locale SET NOT NULL;
```

Comment:
Do we want to default this one to `en`?

Reply:
And if we default this to `en` we don't need the `EMAILS_DEFAULT_LOCALE` env var.

Reply:
Ok. better to have `EMAILS_DEFAULT_LOCALE` as an env var. `EMAILS_DEFAULT_LOCALE` will default to `en` if not set.